src/mm-progress-bar/mm-progress-bar.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @license
+ * Copyright (c) 2015 MediaMath Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at http://mediamath.github.io/strand/LICENSE.txt
+
+*/
+(function(scope) {
+
+	scope.ProgressBar = Polymer({
+		is: 'mm-progress-bar',
+
+		behaviors: [
+			StrandTraits.Stylable
+		],
+
+		properties: {
+			ver: {
+				type: String,
+				value: '<<version>>'
+			},
+			backgroundColor: {
+				type: String,
+				value: Colors.A1
+			},
+			fillColor: {
+				type: String,
+				value: Colors.D3
+			},
+			height: {
+				type: Number,
+				value: 12
+			},
+			indeterminate: {
+				observer: '_updateClass',
+				reflectToAttribute: true,
+				type: Boolean,
+				value: false
+			},
+			percentComplete: {
+				observer: '_clampPercentComplete',
+				type: Number,
+				value: 0
+			},
+			secondaryFillColor: {
+				type: String,
+				value: Colors.D5
+			},
+			width: {
+				type: Number,
+				value: 250
+			}
```

Comment:
shouldn't we just set width/height on the :host css? having something that could responsively adapt to its parent container would make sense here.. instead of having to set defined attributes. For instance.. i want a progress bar that spans the width of a <div>.